README.rst

```diff
@@ -237,21 +239,21 @@ Install your forked repo:
 
     git clone git://github.com/<your_fork>/openwisp-utils
     cd openwisp-utils/
-    python setup.py develop
```

Comment:
@nemesisdesign said: we should replace the line `python setup.py develop` with `pip install -e .[users]`, which will automatically install openwisp-users, which depends on django and will install the right version of django
Did you take this opinion into account?

Reply:
No

Reply:
@h0n3yc4k3 is right

Reply:
ok, will test it on a new virtualenv and make the necessary commits

Reply:
@nemesisdesign @h0n3yc4k3 the `pip install -e .[users]` fetched Django 2.0 as well

Reply:
Yes, default installing on python2 produce error **AttributeError: 'module' object has no attribute 'lru_cache'** because it try to install django2 by default (which not supported).

But you can add note like "If you are using python2 then you shoud run `pip install django~=1.0` (or any 2>django>=1.8) before `pip install -e .[users]`" (for bypassing installing django2)
I didnt check comptability, but it installed :D I'll check it